doc/content/gateways/multitechconduit-mlinux/_index.md

```diff
@@ -0,0 +1,141 @@
+
+We have included a few short steps to help you get started.
+
+### Connecting to the Gateway as a DHCP Server
+
+Since the gateway initializes as a DHCP server, you may have to configure TCP/IP manually. After hard resetting the device, we connected it directly to a computer via ethernet and used the following TCP/IP settings:
+
+- IP Address: 192.168.2.2
+- Subnet Mask: 255.255.255.0
+- Router Address: 192.168.2.1
+
+### Commissioning
+
+After connecting to the gateway, it must be commissioned, or you will not be able to login to the gateway. See instructions [here](https://www.multitech.net/developer/software/mlinux/getting-started-with-conduit-mlinux/commissioning-for-mlinux-devices/). In short, open a browser to the gateway's IP, 192.168.2.1, and set a username and password.
+
+### Configuring the Gateway as a DHCP Client
+
+To connect the gateway to your network, you must reconfigure it as a DHCP client. After commissioning the device, ssh in using the username and password you set up:
+
+```bash
+$ ssh username@192.168.2.1
+```
+
+Modify the network configuration file, `/etc/network/interfaces`:
+
+```bash
+$ sudo nano /etc/network/interfaces
+```
+
+Configure the eth0 peripheral as dhcp, and comment out the static configuration:
+
+```bash
+# Wired interface
+auto eth0
+iface eth0 inet dhcp
+#iface eth0 inet static
+#address 192.168.2.1
+#netmask 255.255.255.0
+#gateway 192.168.2.254
+```
+
+Reboot your gateway to enable the new network configuration, and now you can connect your gateway to your router and finish the configuration for {{% tts %}}.
+
+## Configuration using a Terminal
+
+### Downloading Global_conf.json
+
+Firstly, you will need to generate a `global_conf.json` file required to add a custom channel plan to your gateway. Follow the steps below to generate the required json file.
+
+> **Warning:** The manual configuration of the frequency plan - required to make it work on the The Things (Enterprise) Stack - may result in a mismatch for some of the frequency bands used for the Multitech Gateway.
+
+The Gateway Configuration Server can be used to generate a proper `global_conf.json` configuration file for your gateway. You will need a Gateway API key with the `View gateway information` right enabled. The instructions for the same can be found in the [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}) guide.
+
+Open the command prompt in Windows or any Linux terminal to run a curl command (as shown below) to generate the required `global_conf.json` file in your current working directory.
+
+Make sure you replace `thethings.example.com` with your server address:
+
+```bash
+$ curl -XGET \
+    "https://thethings.example.com/api/v3/gcs/gateways/{GATEWAY_ID}/semtechudp/global_conf.json" \
+    -H "Authorization: Bearer {GTW_API_KEY}" > ~/global_conf.json
+```
+
+> Note: Replace the required fields in the above command and run it.
+
```

Comment:
Can we add the instructions (or reference) for downloading the global_conf.json file from the console?

Reply:
@benolayinka did you look into this?